doc/source/whatsnew/v1.3.0.rst

```diff
@@ -341,7 +341,7 @@ Plotting
 ^^^^^^^^
 
 - Bug in :func:`scatter_matrix` raising when 2d ``ax`` argument passed (:issue:`16253`)
--
+- Support matplotlib's ``constrained_layout=True`` (:issue:`25261`)
```

Comment:
can you be slightly more verbose here. this allows *passing* this option right?

Reply:
The change just prevents pandas from doing incompatible things when a figure has constrained_layout enabled.